src/components/common/MainHeader/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
```suggestion
import { string } from 'prop-types';
```

And then remove line 12